#
# This file is autogenerated by pip-compile with Python 3.8
# by the following command:
#
#    pip-compile --resolver=backtracking ./charms/jupyter-ui/requirements-unit.in
#
attrs==22.1.0
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   jsonschema
    #   pytest
certifi==2022.9.24
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   requests
chardet==3.0.4
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   requests
coverage==6.5.0
    # via -r ./charms/jupyter-ui/requirements-unit.in
exceptiongroup==1.0.4
    # via pytest
idna==2.10
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   requests
iniconfig==1.1.1
    # via pytest
jsonschema==3.2.0
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   serialized-data-interface
oci-image==1.0.0
    # via -r ./charms/jupyter-ui/requirements.txt
ops==1.5.4
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   serialized-data-interface
packaging==21.3
    # via pytest
pluggy==1.0.0
    # via pytest
pyparsing==3.0.9
    # via packaging
pyrsistent==0.19.2
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   jsonschema
pytest==7.2.0
    # via -r ./charms/jupyter-ui/requirements-unit.in
pyyaml==5.4
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   ops
    #   serialized-data-interface
requests==2.25.0
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   serialized-data-interface
serialized-data-interface==0.3.5
    # via -r ./charms/jupyter-ui/requirements.txt
six==1.16.0
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   jsonschema
tomli==2.0.1
    # via pytest
urllib3==1.26.13
    # via
    #   -r ./charms/jupyter-ui/requirements.txt
    #   requests

# The following packages are considered to be unsafe in a requirements file:
# setuptools
